sdk/node/src/api/transactions.js

```diff
@@ -385,6 +397,27 @@ const transactionsAPI = (client) => {
     },
 
     /**
+     * sign - Sign a single transaction.
+     *
+     * @param {Object} template - A single transaction template.
+     * @param {objectCallback} [callback] - Optional callback. Use instead of Promise return value as desired.
+     * @returns {Object} Transaction template with all possible signatures added.
+     */
+    sign: (template, cb) => finalize(template)
```

Comment:
Just a heads-up. When `finalize` does come in, we're going to have to decide how to handle batch errors. If we do it like multiple HSMs, we'll need to add code that distinguishes between errors that occur in the finalize step, and the errors that occur in the sign step.